src/lib.rs

```diff
@@ -105,25 +105,40 @@ mod section;
 pub const RECOMMENDED_SECTION_SIZE: usize = 10;
```

Comment:
Perhaps we need to increase this as well now?

Reply:
Good point. What we be a good value? Maybe `2 * ELDER_SIZE` ?

Reply:
yeh perhaps it's better to make it a factor of ELDER_SIZE so we don't leave one of them unchanged when one of them changes.